index.html

```diff
@@ -20,15 +20,15 @@
         <nav class="nav-top">
```

Comment:
- [ ] [OPTIONAL] You can give your website a more suitable title like "Bruck Portfolio"